doc/backups.rst

```diff
@@ -72,4 +72,64 @@ To secure the user, use an extemely restricted permission in ~/.ssh/authorized_k
 Restoring
 ---------
 
-TODO
+How to backup production servers:
+=================================
+Our backup process consists of two steps:
+1- Our current backup script backs up the production instance in the borg_backups repository located in the project path.
+
+2- Afterwards, this borg repository is copied to an ftp server associated to each production instance.
+
+This FTP server is a free service provided by OVH for each dedicated server and must be manually instanciated via the OVH manager.
+
+TODO: Automate enabling the ftp backup server via OVH APIs.
+
+In order to manually backup a production instance, launch the following::
+
+    fab -c {RC_FILE} execute_backup_borg_repository
+
+In order to set a crontab for a production instance, use this command::
+
+    fab -c {RC_FILE} cron_backup_borg_repository
+
+
+These two commands use the following pattern:
+1. Make sure borg is installed.
+2. Put a configuration file on the remote host with the server FTP information.
+3. Creates a backup script from template with the appropriate remote host information and puts it on the remote host.
+4. For the second host, it sets a cron job to automatically backup the instance daily.
+
+
+How to list backups on a production server:
+===========================================
+Use the command::
+
+    fab -c {RC_FILE} list_backups
+
+How to restore a backup on a production instance:
+=================================================
+Use the command::
```

Comment:
Explain RC file variables that have to be set before this step can be run.

Reply:
nont sure what you mean @maparent , at this step, there are no variables to be set except everything that concerns the ftp server, which has been previously explained.

Reply:
those commands assume that `destination_folder`, `borg_password` and `backup_name` are filled in the rc file. Are those variables documented anywhere in the documentation? (Or at worst in `base_env.rc`...)

Reply:
I think they are documented in the base_env file

Reply:
Check again, they are not.

Reply:
actually, it was done in bluenonve.rc